jetty-core/jetty-websocket/jetty-websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/WebSocketClient.java

```diff
@@ -60,7 +60,7 @@ public class WebSocketClient extends ContainerLifeCycle implements Configurable,
     private final SessionTracker sessionTracker = new SessionTracker();
     private final Configuration.ConfigurationCustomizer configurationCustomizer = new Configuration.ConfigurationCustomizer();
     private boolean stopAtShutdown = false;
-    private long _stopTimeout = Long.MAX_VALUE;
+    private long _stopTimeout = 30;
```

Comment:
30 milliseconds?

That seems useless.
How about instead we just set this to 0 or -1 as default.
And if the user wants graceful, they have to call `setStopTimeout(long)` to set a reasonable value.

Reply:
Actually, `Server` has a default value of 0 for `_stopTimeout` so using that value here too would also better align the default behaviors across components.
(obviously, it was supposed to be 30s)